### PR TITLE
Add network info to Mac VM logs

### PR DESCRIPTION
**Pull request by ThrasherLT**
### Problem
It is currently difficult to reproduce LLT-4600 and a couple of more Mac VM related flaky NAT-Lab tests.

### Solution
Added network interface name and IP, routing table and the last 20 lines of syslog to a separate log file for MAC VM.

### :ballot_box_with_check: Definition of Done checklist
- [x] Commit history is clean ([requirements](../blob/main/docs/git_commit_messages_requirements.md))
- [x] README.md is updated
- [x] changelog.md is updated
- [x] Functionality is covered by unit or integration tests

### CODE REVIEW

File: nat-lab/tests/telio.py [hunk 1]
```diff
@@ -870,7 +885,7 @@ async def save_logs(self) -> None:
         os.makedirs(log_dir, exist_ok=True)
 
         log_content = await self.get_log()
-
+        network_info_info = await self.get_network_info()
```
Comment (LukasPukenis) [inline nat-lab/tests/telio.py:888]: I think it would be a slight improvement by moving it nearby where the writing happens

**Review by LukasPukenis — APPROVED** [review]
+2

**Comment by packgron** [comment]
I think together with this let's add system logs.
There is log command in macos, and you can specify --start and --end to see what happened at via time of test's specifically
https://www.manpagez.com/man/1/log/

File: nat-lab/tests/telio.py (hunk 1 above)
Comment (ThrasherLT) [reply nat-lab/tests/telio.py]: Done

**Comment by ThrasherLT** [comment]
> I think together with this let's add system logs. There is log command in macos, and you can specify --start and --end to see what happened at via time of test's specifically https://www.manpagez.com/man/1/log/

Done

**Review by packgron — APPROVED** [review]

**Pull request merged**